pkg/rebuild/maven/strategy.go

```diff
@@ -42,8 +43,6 @@ func (b *MavenBuild) ToWorkflow() (*rebuild.WorkflowStrategy, error) {
 				Uses: "maven/export-java",
 			},
 			{
-				// TODO: Java 9 needs additional certificate installed in /etc/ssl/certs/java/cacerts
-				// It can be passed to maven command via -Djavax.net.ssl.trustStore=/etc/ssl/certs/java/cacerts
```

Comment:
I have not added the flow steps to Maven yet because I only saw them on Gradle builds as of now. But it should be pretty straightforward.